src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -0,0 +1,10 @@
+export type Organization = {
+  id: string;
+  slug: string;
+  projects: any[];
```

Comment:
should this be Project[]?

Reply:
I'll try and get this to working with Project[] in a bit, right now it won't since we have some extra properties on the project here

Reply:
Hopefully we don't have projects + teams at all before the end of the year

Reply:
wait... really? what's happening with projects and teams?

Reply:
@billyvg Updated to Project[]

Reply:
We need to separate it from org details to improve perf for large orgs